### ISSUE TITLE
Landing Page: 'Blank' surveys

### ISSUE THREAD

**Issue by adrianq** [EyeSeeTea/SurveillanceCambodiaApp#31]
Feedback from qispsk@gmail.com : I believe this issue has already been raised, but curious about empty data being sent to the server. It looks like there are a lot of N/A survey data in DHIS and it throws off the monitor module as these show up as suspected cases. Also, I believe these two particular surveys may be due to surveys that were completed before I input the correct organizational unit - though I haven't been able to test this theory.

 [Link to buddybuild feedback from build 11](https://dashboard.buddybuild.com/apps/56d792f0e4d6010100721090/feedback?fid=56e931cb9492a3010040a903&bnum=11)

<img src="https://s3-us-west-2.amazonaws.com/buddybuild-screenshots/56d792f0e4d6010100721090/56e8c3568418ae01009093f9/039fbb29-b356-4a0c-a4be-288c3032f465.jpg" width="33%" height="33%" /><table><tr><td>Build</td><td>11</td></tr><tr><td>Device type</td><td>LG-H324</td></tr><tr><td>Screen size</td><td>480</td></tr><tr><td>Screen size</td><td>480px by 786px</td></tr><tr><td>Battery</td><td>38% Unplugged</td></tr><tr><td>Memory free</td><td>192 MB / 711 MB</td></tr></table>

**Comment by adrianq** [comment]
Yes, it has although it was slightly different. Anyway we were not able to reproduce the problem.  Definitively, we need to catch it. @QISPSK can you let me know what server/org unit you  had configured in the settings?

**Comment by gtrsmith** [comment]
Is this not due to answering either '-' or 'x' to the first survey question? I think the NA survey data in DHIS is likely due to this.

**Comment by QISPSK** [comment]
@adrianq @gtrsmith 
Server and org unit were not set initially when two surveys were completed, but were later set to server: https://www.psi-mis.org and org unit: KHTest. A coworker mentioned that submitting surveys before the org unit is set or changing the date/time before a survey is set can result in blank surveys being sent. I tested both of these options and they didn't seem to have an impact in my case, however the application did freeze at one point when I tried to select "not tested" and it looks like another blank survey was sent.
